### DISCUSSION TITLE
Full width cards in slider?

### DISCUSSION THREAD

**Discussion by poneilcsgp** [microsoft-search/pnp-modern-search#2168] [Q&A]
I'm trying to figure out if I can use a full width 'card' in a slider based search result template.   I see that Flickity is the component that the slider is based off of and Flickity docs have examples of this,  but I can't figure out if this is possible  w custom templates based on search results slider.

**Comment by wobba** [comment]
You should be able to pass in options.

```html
<pnp-slider data-items="{{JSONstringify @root.data.items 2}}" 
            data-options="{{JSONstringify @root.properties.layoutProperties.sliderOptions}}"
```

Default options are: 
```ts
            autoPlay: true,
            autoPlayDuration: 3,
            pauseAutoPlayOnHover: true,
            numberOfSlides: 3,
            showPageDots: true,
            wrapAround: false,
```

**Reply by poneilcsgp** [reply]
Yes I've been playing with those, but it looks like slide width is set on the slide element itself and specifying relative units (e.g. 100%) doesn't seem to work.   When comparing to the flickity example (which does use relative units):   https://codepen.io/desandro/pen/dPdyJK  it appears that the SPFx part renders two additional divs under the &lt;div class="flickity-slider"&gt; element, but I'm not sure if that is the issue.

**Reply by poneilcsgp** [reply]
I think I figured it out.   By adding a CSS class .template div{ width: 100%; } to enforce 100% on all divs that get rendered, then I can set the slide width as a percentage.
